sw/device/silicon_creator/rom/keys/README.md

```diff
@@ -0,0 +1,27 @@
+## Re-generating ROM keys
+
+The following steps generate a new fake test SPHINCS+ key pair and update the
+various places that reference it. The instructions can be modified to generate
+other types of keys:
+- To generate ECDSA keys, replace every occurrence of `spx` with `ecdsa`.
+- To generate prod or dev keys, replace every occurrenct of `test_key` with `prod_key` or `dev_key`.
+- To generate real keys, use the `keys/real` directory instead of `keys/fake`.
```

Comment:
nit: For real keys we recommend using an offline HSM, so this procedure is only applicable to fake keys.

Reply:
Do we have a way of generating those keys ahead of time and checking in simulation that the chip definitely can boot using them, so that we don't end up with a subtle encoding mismatch between the HSM and the ROM?